controller/iteration_blackbox_test.go

```diff
@@ -612,21 +611,22 @@ func (rest *TestIterationREST) createIterations() (*app.IterationSingle, *accoun
 	return created, owner
 }
 
-// TestIterationActivedByUser tests iteration should always be active when user sets it to active
+// TestIterationActivedByUser tests:
```

Comment:
Typo. Should be `TestIterationActivatedByUser`

Reply:
fixed it here e4ffadb